File: src/main/java/me/kuku/yuq/service/WeiboService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.WeiboEntity;

import java.util.List;

@AutoBind
public interface WeiboService {
    WeiboEntity findByQQ(Long qq);
    void save(WeiboEntity weiboEntity);
    int delByQQ(Long qq);
    List<WeiboEntity> findByMonitor(Boolean monitor);
    List<WeiboEntity> findAll();
}

File: src/main/kotlin/me/kuku/yuq/service/WeiboService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.WeiboEntity

@AutoBind
interface WeiboService {
    fun findByQQ(qq: Long): WeiboEntity?

    fun save(weiboEntity: WeiboEntity)

    fun delByQQ(qq: Long): Int

}